x-pack/platform/plugins/shared/index_management/test/scout/ui/tests/index_template_wizard.spec.ts

```diff
@@ -0,0 +1,128 @@
+    // Fill logistics step
+    await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();
+
+    // Skip component templates step
+    await pageObjects.indexManagement.clickNextButton();
+
+    // Verify step title
+    await expect(page.testSubj.locator('stepTitle')).toHaveText('Index settings (optional)');
+
+    // Verify that index mode callout is displayed
+    await expect(page.testSubj.locator('indexModeCallout')).toHaveText(
+      'The index.mode setting has been set to Standard within the Logistics step. Any changes to index.mode set on this page will be overwritten by the Logistics selection.'
+    );
+
+    // Click Next button
+    await pageObjects.indexManagement.clickNextButton();
+  });
+
+  test('renders mappings (step 4)', async ({ page, pageObjects }) => {
+    // Fill logistics step
+    await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();
+
+    // Skip component templates and settings steps
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+
+    // Verify step title
+    await expect(page.testSubj.locator('stepTitle')).toHaveText('Mappings (optional)');
+
+    // Click Next button
+    await pageObjects.indexManagement.clickNextButton();
+  });
+
+  test('renders aliases (step 5)', async ({ page, pageObjects }) => {
+    // Fill logistics step
+    await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();
+
+    // Skip component templates, settings, and mappings steps
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+
+    // Verify step title
+    await expect(page.testSubj.locator('stepTitle')).toHaveText('Aliases (optional)');
+
+    // Click Next button
+    await pageObjects.indexManagement.clickNextButton();
+  });
+
+  test('renders review template (step 6)', async ({ page, pageObjects }) => {
+    // Fill logistics step
+    await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();
+
+    // Skip all intermediate steps
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+    await pageObjects.indexManagement.clickNextButton();
+
+    // Verify step title
+    await expect(page.testSubj.locator('stepTitle')).toHaveText(
+      "Review details for 'test-index-template'"
+    );
+
+    // Verify that summary exists
+    await expect(page.testSubj.locator('summaryTabContent')).toBeVisible();
+
+    // Verify that index mode is set to "Standard"
+    await expect(page.testSubj.locator('indexModeTitle')).toBeVisible();
+    await expect(page.testSubj.locator('indexModeValue')).toHaveText('Standard');
+
+    // Click Create template
+    await pageObjects.indexManagement.clickNextButton();
+  });
```

Comment:
_question_: do we really need to repetitively start fresh browser context and repeat UI flow from scratch?

It would be more efficient and pretty much the same outcome to switch from `test` to `test.step` because if  `await pageObjects.indexManagement.indexTemplateWizard.completeStepOne();` doesn't work, there is no point of running most of `test blocks` here. Thoughts?

Reply:
Agreed. Thanks for catching this.